### PR TITLE
Casper/send screen

### PR DESCRIPTION
**Pull request by lawRathod**
Changes: 
* Send transaction confirmation screen shows unit and microunit. 
* Refac on mobile TransactionConfirmFields.tsx

### CODE REVIEW

File: libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts [hunk 1]
```diff
@@ -31,14 +20,14 @@ function getDeviceTransactionConfig({
 }): Array<DeviceTransactionField> {
   const fields: Array<DeviceTransactionField> = [];
   fields.push({
-    type: "casper.amount",
+    type: "casper.extendedAmount",
     label: "Amount",
-    value: `${transaction.amount.toNumber().toLocaleString()} motes`,
+    value: transaction.amount.toString(),
```
Comment (emmanuelm41) [inline libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts]: We should use "toFixed" as "toString" has some issues when using long long numbers.

File: libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts [hunk 2]
```diff
@@ -31,14 +20,14 @@ function getDeviceTransactionConfig({
...
   });
   fields.push({
-    type: "casper.fees",
+    type: "casper.extendedAmount",
     label: "Fees",
-    value: `${transaction.fees.toNumber().toLocaleString()} motes`,
+    value: transaction.fees.toString(),
```
Comment (emmanuelm41) [inline libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts]: Same here.  We should use "toFixed" as "toString" has some issues when using long long numbers.

**Review by emmanuelm41 — CHANGES_REQUESTED** [review]

File: libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts (hunk 1 above)
Comment (lawRathod) [reply libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts]: Ohh, that's insightful. Done!

Comment (emmanuelm41) [reply libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts]: Just for reference 
https://github.com/MikeMcl/bignumber.js/issues/209

**Pull request merged**